webpack.common.js

```diff
@@ -98,6 +99,7 @@ function createConfig(target /*: Target */, dev /*: boolean */,
             !dev && new MiniCssExtractPlugin({
                 filename: minimize ? '[name].min.css' : '[name].css',
             }),
+            new webpack.optimize.ModuleConcatenationPlugin(),
```

Comment:
Does `ModuleConcatenationPlugin` reduce build size?

Reply:
I tried this, but it bails out.  See my comments in https://github.com/Khan/KaTeX/issues/872.